package.json

```diff
@@ -25,7 +25,7 @@
   },
   "homepage": "https://github.com/mrq-cz/slackify-html",
   "dependencies": {
-    "html-entities": "^1.1.3",
+    "html-entities": "^2.3.3",
```

Comment:
I think this major upgrade make break this? I tried running the tests and I get this 

<img width="968" alt="Screenshot 2023-04-27 at 2 12 29 PM" src="https://user-images.githubusercontent.com/26090238/234954834-ca4c88f7-845f-42aa-8ff9-1cd421648dec.png">

Reply:
That's odd. Tests pass just fine for me. I just switched to the `fullhtml` branch locally, merged mine, and ran `npm install && npm test` and it runs fine as well.

Reply:
`decode` is definitely a function in `html-entities` so something seems off there.

Reply:
ah forgot to install my b